ironfish-cli/src/commands/wallet/mint.ts

```diff
@@ -97,6 +97,12 @@ This will create tokens and increase supply for a given asset.`
     transferOwnershipTo: Flags.string({
       description: 'The public address of the account to transfer ownership of this asset to.',
     }),
+    transferTo: Flags.string({
+      description: 'transfer all newly minted coins to this public address.',
```

Comment:
```suggestion
      description: 'Transfer all newly minted coins to this public address.',
```

Reply:
Descriptions for flags should all be lower case since the CLI audit.

From our style guide...

> Ensure that descriptions are provided for all flags, that the descriptions are in lowercase, that they are concise (so as to fit on narrow screens), and that they do not end in a period to match other flag descriptions.